### PR TITLE
Fixes #20. 'template' property is deprecated use 'view' property.

### LINKED ISSUE #20: Change template to view
In 3.0 `template` is deprecated ([source](https://github.com/canjs/can-component/blob/master/can-component.js#L104)) so the generated code should use `view` instead.

### PR DESCRIPTION
**Pull request by mjstahl**

### CODE REVIEW

File: test/component_test.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ test("Defines the correct loader", function(){
 	myLoader.import(myLoader.configMain).then(function(){
 		return myLoader.import("test/tests/frankenstein.component!");
 	}).then(function(){
-		var template = defines["test/tests/frankenstein.component-template"];
+		var template = defines["test/tests/frankenstein.component-view"];
```
Comment (marshallswain) [inline test/component_test.js]: @matthewp I imagine that we still want to use `<template>` tags, but do we want this part to be `component-view`?

**Comment by marshallswain** [comment]
This maybe gets the tests a little further along.  :)

**Comment by marshallswain** [comment]
Looking at the last Travis build, it looks like the next step is to replace can-ssr with done-ssr.  can-ssr is trying to load  `ENOENT, open '/home/travis/build/donejs/done-component/test/tests/ssr/can/view/stache/stache.js'`

**Comment by marshallswain** [comment]
@phillipskevin why was this closed?

**Comment by phillipskevin** [comment]
Oops. I moved the `can3` branch over to `major. Reopening pointing to `major`.

**Comment by marshallswain** [comment]
We ought to add backwards compatibility so that `<template></template>` tags still work.

**Comment by matthewp** [comment]
Working on this now.

**Pull request merged**